.github/workflows/woodwork_main_performance_tests.yaml

```diff
@@ -0,0 +1,143 @@
+name: Run looking glass performance tests
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  performance_tests:
+    name: Run looking glass performance tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_SECRET_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: 'us-east-1'
+      AWS_SESSION_TOKEN: ""
+    steps:
+      - name: Checkout woodwork
```

Comment:
This default checks out `main` right? So `git rev-parse --short HEAD~1` gets the previous commit on `main`?